parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -127,7 +127,7 @@ EOF
 }
 
 configureHTTPProxyCA() {
-    if [[ $OS == $MARINER_OS_NAME ]]; then
+    if [[ $OS == $MARINER_OS_NAME ]] || [[ $OS == $AZURELINUX_OS_NAME ]]; then
```

Comment:
this will now also run on both mariner V3 and mariner V2 based AzureLinux SKUs, is that intended?

Reply:
I believe this was running on the mariner V2 based AzureLinux SKU already as `$OS` is MARINER for the mariner V2 based AzureLinux SKU as well. Now, it will also run on mariner V3